src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Black Student Insight</a></li> 
+          <li><a href="college-info.html">Find a School</a></li>
+        <!-- <li><a href="contact-us.html">Contact Us</a><li>  -->
+          <!-- future contact page? -->
+          </ul>
+        <input type="search" id="school-search" onsearch="loadSchoolInfo()" 
+        placeholder="Find a School" onkeypress="hideElements()"/>
+        <!-- <home-button onsearch="loadSchoolInfo()">Search</home-button> -->
+      </nav> 
+    </header>
+    
+    <div id= "information">
+      
+      <h2>Our Focus</h2>
+      <br>
+      <p>Black Student Insight serves as a platform for current minority students 
+          to share their experiences and beliefs regarding their campuses, but also 
+          allows prospective students to look at diversity breakdowns, as well as
+          average admissions statistics for their university of interest.
+      </p>
+      <br>
```

Comment:
For good practice, remove all of the `<br>` and add space through the margins and padding instead.